docs/Maintainers.md

```diff
@@ -13,6 +13,15 @@ but when that fails, the Committee calls for a vote where the super majority
 (two-thirds) wins. This is to prevent obstructionism by removing the possibility
 of a one person veto.
 
+Quorum for Community Maintenance Committee meetings requires at least two-thirds
```

Comment:
Depending on PR ordering, we may have to change this over to Governance.

Reply:
Updated name